src/theme/CodeBlock/Line/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
no TS?

Reply:
Sadly not. This file is generated by running a Docusaurus command. Docusaurus will then use this component instead of the native one for all code blocks. I guess Docusaurus doesn't support TS for their internals.